### PR TITLE
fix issue #288

### PR DESCRIPTION
**Pull request by kinfkong**
fix issue #288

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -307,7 +307,6 @@ export class AppComponent implements OnInit {
    * @param err is command added failed
    */
   onCommandAdded(err) {
-    this.onRefresh();
```
Comment (sumitdaga) [inline src/app/app.component.ts]: @kinfkong  
i am not sure but it should refresh if add command is a success ..isn't it?

Comment (sumitdaga) [reply src/app/app.component.ts]: @standlove

Comment (standlove) [reply src/app/app.component.ts]: Yes, if the data on the UI is affected, the new data should be reloaded.

Comment (kinfkong) [reply src/app/app.component.ts]: Yes, you are right. However the current refresh doesn't work, check:
https://monosnap.com/file/sdMjNLx4UZSA2ywjiSEa7RTVL9Wkbg
maybe this is related another bug.

**Comment by standlove** [comment]
yes, there is another bug for this. I will close this.

**Pull request closed without merging**